docs/als/settings.md

```
@@ -14,6 +14,11 @@ Toggle usage of fully qualified collection names (FQCN) when inserting module na
 . Default value:
 ``true``
 
+## [`ansible.config.path`](#config.path) { #config.path data-toc-label=config.path }
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)